src/parser/jobs/mnk/modules/Brotherhood.tsx

```diff
@@ -0,0 +1,89 @@
+import {t, Trans} from '@lingui/macro'
+import {ActionLink} from 'components/ui/DbLink'
+import {StatusRoot} from 'data/STATUSES'
+import {Event, Events} from 'event'
+import {Analyser} from 'parser/core/Analyser'
+import {filter, oneOf} from 'parser/core/filter'
+import {dependency} from 'parser/core/Injectable'
+import Checklist, {Requirement, Rule} from 'parser/core/modules/Checklist'
+import {Data} from 'parser/core/modules/Data'
+import React from 'react'
+import {DISPLAY_ORDER} from './DISPLAY_ORDER'
+
+const FULL_PARTY_SIZE = 8
+
+export class Brotherhood extends Analyser {
+	static override debug = false
+	static override handle = 'brotherhood'
+	static override title = t('mnk.bh.title')`Brotherhood`
+	static override displayOrder = DISPLAY_ORDER.BROTHERHOOD
+
+	@dependency private checklist!: Checklist
+	@dependency private data!: Data
+
+	private brotherhoodBuffTimestamps: number[] = []
+	private brotherhoodCastTimestamps: number[] = []
+	private brotherhood: StatusRoot['BROTHERHOOD'] = this.data.statuses.BROTHERHOOD
+
+	override initialise() {
+		super.initialise()
+
+		const playerCharacters = this.parser.pull.actors
+			.filter(actor => actor.playerControlled)
+			.map(actor => actor.id)
+
+		this.addEventHook(filter<Event>()
+			.type('statusApply')
+			.status(this.data.statuses.BROTHERHOOD.id)
+			.source(this.parser.actor.id)
+			.target(oneOf(playerCharacters)),
+		this.onBrotherhoodBuff)
+
+		this.addEventHook(filter<Event>()
+			.type('action')
+			.action(this.data.actions.BROTHERHOOD.id)
+			.source(this.parser.actor.id),
+		this.onBrotherhoodAction)
+
+		this.addEventHook(filter<Event>().type('complete'), this.onComplete)
+	}
+
+	private onComplete() {
+		this.checklist.add(new Rule({
+			name: <Trans id="mnk.bh.checklist.name">Buff your party</Trans>,
+			description: <Trans id="mnk.bh.checklist.desc">
+				To maximise raid damage, try to hit all party members with <ActionLink action="BROTHERHOOD"/>.
+			</Trans>,
+			displayOrder: DISPLAY_ORDER.BROTHERHOOD,
+			requirements: [
+				new Requirement({
+					name: <Trans id="mnk.bh.checklist.req.name"><ActionLink action="BROTHERHOOD"/> buffs</Trans>,
+					value: this.totalAffectedPlayers(),
+					target: this.maxPossibleTargets(),
```

Comment:
TBH the target is probably 100% unless the other players are dead.

Reply:
I dont think so since Brotherhood has a 15 yard range
![image](https://user-images.githubusercontent.com/17149559/154400454-4627334d-cde8-4a82-be00-6dfab00c904a.png)
I also think it's good to set a target so the person can look at the log and ask themselves why didn't they hit that target of the max possible.

Reply:
If the target is always 100%, then it's clear that someone was out of range. The target here sets at what point the checklist turns green, so if they see green with say 50% because all the ranged/casters are in Narnia, they won't necessarily notice the error and get their party to stay in range. Only reason I can see to adjust this is for death, since that's not something the player can control, although even then they could try telling their party to not die.